0-RegEx.md

```diff
@@ -121,4 +154,6 @@ public class Part
 ```c#
 answer:
 
+(public\sclass\s\w*)\n{\n\s*public\sstring\s\w*\s{\sget;\sset;\s}\n\s*public\sint\s\w*\s{\sget;\sset;\s}\n}
```

Comment:
This doesn't capture the property names but it does match the whole class. It's on the right track.